etc/listr2.api.md

```diff
@@ -233,7 +233,7 @@ export class ListrTaskObject<Ctx, Renderer extends ListrRendererFactory> extends
     // (undocumented)
     options: ListrOptions;
     // (undocumented)
-    set output$(data: string);
+    set output$(data: string | undefined);

```

Comment:
backwards compatible because those who were assigning `output$` to a string may still do so